week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,9 @@
+function percentage(a, b) {
```

Comment:
for code quality and reduce the effort needed to read and understand your code always try to give a meaning names and avoid using letters, numbers, so can you update the parameters and make it describe the parameter.